ReactAndroid/src/main/java/com/facebook/react/uimanager/NativeViewHierarchyManager.java

```diff
@@ -218,11 +229,7 @@ public void createView(
       View view = viewManager.createView(themedContext, mJSResponderHandler);
       mTagsToViews.put(tag, view);
       mTagsToViewManagers.put(tag, viewManager);
-
-      // Use android View id field to store React tag. This is possible since we don't inflate
-      // React views from layout xmls. Thus it is easier to just reuse that field instead of
-      // creating another (potentially much more expensive) mapping from view to React tag
-      view.setId(tag);
+      view.setTag(tag);
```

Comment:
nit: The existing comment might be useful to keep around:

Since we don't need Android tags for anything else, reuse the field instead of creating another (potentially much more expensive) mapping from view to React tag.